x-pack/legacy/plugins/siem/public/pages/case/components/configure_cases/index.tsx

```diff
@@ -73,20 +100,16 @@ const ConfigureCasesComponent: React.FC = () => {
     setConnectorId,
     setClosureType,
   });
-  const {
-    loading: isLoadingConnectors,
-    connectors,
-    refetchConnectors,
-    updateConnector,
-  } = useConnectors();
+  const { loading: isLoadingConnectors, connectors, refetchConnectors } = useConnectors();
 
+  const reloadConnectors = useCallback(async () => refetchConnectors(), []);
```

Comment:
do we need `async`?

Reply:
maybe we could return memoized `refetchConnectors`, so we don't need to do it here?

Reply:
`ActionsConnectorsContextProvider` needs an `async` function so we do this to make it happy.